### ISSUE TITLE
Clean up "Create Packing Slip" modal

### ISSUE THREAD

**Issue by jarrilla** [conturo-prototyping/pack-ship-demo#66]
- [x] Pack Qty cells should be auto-matically populated with `fulfilledQty > batchQty ? 0 : batchQty - fulfilledQty`
- [x] Remove grid footer (the whole rows per page thing.. make the whole thing one big scrollable area)
- [ ] ~Pack Qty cells should be very obviously editable (i.e. no need to double click; they should feel more like native inputs)~
- [ ] ~When first opening modal, focus first row's Pack Qty input~

**Comment by jbolles94** [comment]
@jarrilla After spending a little time investigating this, Material UI's DataGrid (which we use for rendering the tables) does not have support for changing the double click of a cell to a single click. In addition, it does not support autofocusing the first element. I can certainly change the styling of it to look more editable (I have attached an example that can be tweaked if desired).  If we want the single click and autofocusing functionality, I will have to remove our use of the DataGrid component for the CreatePackingSlip and EditPackingSlip tables and switch them to use [Material UI's table component](https://mui.com/api/table/) which will take a decent amount of refactoring since it requires a lot more setup of the children jsx elements. Before I did this though, I wanted your thoughts to see if it was worth spending the time refactoring those elements to get the desired look and feel. What are your thoughts? I have already completed tasks two and three of this issue if that in itself is worth pushing up.      


![image](https://user-images.githubusercontent.com/91438599/152690842-9cbadccc-a4bc-4f79-b648-3def796b11ea.png)
